src/System.Management.Automation/engine/CmdletParameterBinderController.cs

```diff
@@ -1267,7 +1183,7 @@ private static bool IsParameterScriptBlockBindable(MergedCompiledCommandParamete
         /// </param>
         internal override Collection<CommandParameterInternal> BindParameters(Collection<CommandParameterInternal> parameters)
         {
-            return BindParameters(uint.MaxValue, parameters);
+            throw new NotImplementedException();
```

Comment:
Since this is internal access, why throw an exception here?  Shouldn't the method just be removed from the abstract class?  If not, please add comment as to why this is not implemented.

Reply:
Good point. Will move the 'NotImplementedException' to be base method.